src/pykeen/cli.py

```diff
@@ -181,6 +185,42 @@ def _format_reference(reference: Optional[str], link_fmt: Optional[str], alt_ref
     return f"[`{reference}`]({link_fmt.format(alt_reference or reference)})"
 
 
+def _get_resolver_lines2(resolver: ClassResolver, link_fmt: str) -> Iterable[Tuple[str, Optional[str]]]:
+    for _, cls in sorted(resolver.lookup_dict.items()):
+        reference = f"{cls.__module__}.{cls.__qualname__}"
+        docdata = getattr(cls, "__docdata__", {})
```

Comment:
I think you want `alt_reference = resolver.docdata(cls, "name")`

see https://class-resolver.readthedocs.io/en/latest/api/class_resolver.ClassResolver.html#class_resolver.ClassResolver.docdata